content/synthetics/browser_check.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Browser Check
+kind: documentation
+beta: true
+description: Simulate and monitor user journeys from specific locations.
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/uptime_check"
+  tag: "Documentation"
+  text: "Configure an Uptime Check"
+---
+
+<div class="alert alert-warning">Synthetics is in beta for the Datadog US Site. Request access via the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics request form</a>.</div>
+
+## Overview
+
+Browser Checks are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These checks verify both that your applications are up and responding to requests, and that any conditions defined in your scenarios are met.
+
+## Configuration
+
+### Test details
+
+Define the configuration of your Browser Check.
+
+{{< img src="synthetics/browser_check/browser_check_configuration.png" alt="Browser Check make request" responsive="true" style="width:80%;">}}
+
+1. Give your Browser Check a name. 
+2. Enter the Start URL. This is the URL from which your Browser Check starts the scenario.
+3. Define on which device to run your check. Available devices are: `Laptop Large`, `Tablet`, and `Mobile Small`.
+4. Pick-up locations to run the test from. Available locations are:
+    * Frankfurt (Request made from an AWS Datacenter)
+    * Tokyo (Request made from an AWS Datacenter)
+5. Choose a Check frequency between "1 run per 5 minute interval" to "1 run per week":
+
+    {{< img src="synthetics/browser_check/check_frequency.png" alt="Check frequency" responsive="true" style="width:80%;">}}
+
+### Notifications
+
+A notification is sent if at least one step of the Browser Check scenario fails. To configure your notifications:
+
+{{< img src="synthetics/browser_check/browser_check_notification.png" alt="Browser check notification" responsive="true" style="width:80%;">}}
+
+1. Select users and/or [services][1] to send the notifications to.
+2. Enter a **message** for the Browser Check. This field allows standard [Markdown formatting][2]. Notification messages include the **message** defined in this section and information about which assertion failed and why.
+3. Click **Save Details and Record Test** to save your Browser Check.
+4. Start to record your test.
```

Comment:
Don't know if it worth explaining that, but the process is
- when a check is created for the first time, it's created *paused*
- the first time you click "Save Details and Record Test", you go to the test recorder page in a special mode where you can "Save And Launch" which will save the steps and un-pause the check
- the other time you go to the test recorder page, you can only "Save" and "Save And Quit" which save steps but won't change the status pause / live of the check.

Also something subtle is that, if a check is live but with zero steps, we'll still run it by adding ourself a single step which is the navigation to the start URL of the check. (This is indicated by a warning in the check details page).

Reply:
no need to complexify the doc here, the DD UI is clear enough about this.